src/pages/ForgotPasswordV2.jsx

```diff
@@ -0,0 +1,144 @@
+import {
+    Box,
+    Button,
+    Center,
+    FormControl,
+    FormLabel,
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/latest/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'FormLabel' is defined but never used.